Description: drop tun_tap dependency and its related tests
 tun_tap still uses a bunch of old tokio crates that are all now
 incorporated into tokio. This could mean a lot of patching, so for
 the sake of sanity temporarily just disabling the depndency since
 it test only and also disabling the tests that use tun_tap
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: no
Last-Update: 2024-10-13
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -113,5 +113,5 @@
 
-[[test]]
-name = "tap_tests"
-path = "tests/tap_tests.rs"
+#[[test]]
+#name = "tap_tests"
+#path = "tests/tap_tests.rs"
 
@@ -144,4 +144,4 @@
 
-[dev-dependencies.etherparse]
-version = "0.13.0"
+#[dev-dependencies.etherparse]
+#version = "0.13.0"
 
@@ -173,4 +173,4 @@
 
-[target.'cfg(not(target_os = "windows"))'.dev-dependencies.tun-tap]
-version = "0.1.3"
+#[target.'cfg(not(target_os = "windows"))'.dev-dependencies.tun-tap]
+#version = "0.1.3"
 
